tests/pass/backtrace/backtrace-api-v0.stdout

```diff
@@ -1,5 +1,5 @@
 $DIR/backtrace-api-v0.rs:24:14 (func_d)
 $DIR/backtrace-api-v0.rs:20:5 (func_c)
-$DIR/backtrace-api-v0.rs:9:5 (func_b)
+$DIR/backtrace-api-v0.rs:9:5 (func_b::<u8>)
```

Comment:
Fascinating, why does this change?

Reply:
I have absolutely no clue

Reply:
(I think I changed some default path filters and they may affect this but I don't know why)

Reply:
It seems like somehow this filter no longer applies?

https://github.com/rust-lang/miri/blob/7a6a812794f6c2ff46a5a6d1a2991ecc5903b10c/tests/pass/backtrace/backtrace-api-v0.rs#L0-L1

Reply:
Ah wait, that's a stderr filter. Was there a bug previously where those applied to stdout as well?